package.json

```diff
@@ -24,7 +31,6 @@
   "main": "./index.js",
   "module": "./index.mjs",
   "types": "./index.d.ts",
-  "type": "commonjs",
```

Comment:
nodejs documentation encourages to explicitly specify `"type": "commonjs"` for CommonJS modules to be future-proof, currently not explicitly specifying it works exactly the same on nodejs and there are no plans to change that behavior at the moment.
However, this seems to break Webpack's module system resolver at the moment, and until this is fixed at Webpack's end, it would be better to remove it for now.